chart/ory/charts/gcloud-sqlproxy/README.md

```diff
@@ -0,0 +1,145 @@
+| `podDisruptionBudget`             | Pod disruption budget                   | `maxUnavailable: 1` if `replicasCount` > 1, does not create the PDB otherwise               |
+| `service.type`                    | Kubernetes LoadBalancer type            | `ClusterIP`                                                                                 |
+| `service.internalLB`              | Creates service with `cloud.google.com/load-balancer-type: "Internal"` | Default `false`. If `true`, you must also set the `service.type=LoadBalancer`. |
+| `rbac.create`                     | Create RBAC configuration w/ SA         | `false`                                                                                     |
+| `serviceAccount.create` | Create a service account | `true` |
+| `serviceAccount.annotations` | Annotations for the service account | `{}` |
+| `serviceAccount.name` |  Service account name | Generated using the fullname template |
+| `networkPolicy.enabled`           | Enable Network Policy                    | `false`                                                                                     |
+| `networkPolicy.ingress.from`      | List of sources to have access to the Pods selected for this rule. If empty, allows all sources. | `[]`                  |
+| `extraArgs`                       | Additional container arguments          | `{}`                                                                                        |
+| `livenessProbe.enabled`           | Would you like a livenessProbe to be enabled  | `false`                                                                               |
+| `livenessProbe.port`              | The port which will be checked by the probe   | 5432                                                                                  |
+| `livenessProbe.initialDelaySeconds` | Delay before liveness probe is initiated    | 30                                                                                    |
+| `livenessProbe.periodSeconds`     | How often to perform the probe. Provide the interval in seconds.  | 10                                                                                    |
+| `livenessProbe.timeoutSeconds`    | Number of seconds after which the probe times out.  | 5                                                                                     |
+| `livenessProbe.failureThreshold`  | Minimum consecutive failures for the probe to be considered failed after having succeeded.  | 6                                       |
+| `livenessProbe.successThreshold`  | Minimum consecutive successes for the probe to be considered successful after having failed | 1                                       |
+| `readinessProbe.enabled`          | would you like a readinessProbe to be enabled | `false`                                                                               |
```

Comment:
This sounds like a question, but is missing the question mark. Start the sentence with a capital letter.